packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -676,8 +695,36 @@ export function mixinAttach<
 				clients,
 				summarizerClient,
 			};
-		}
+		} else if (operation.type === "rehydrate") {
+			const clientA = state.clients[0];
+			assert.equal(state.clients.length, 1);
+			// This is necessary to get all IdCreationRanges finalized before connecting further clients.
+			// The production codepath also finalizes ids before attaching. It's difficult for the mocks
```

Comment:
nit: the later part of this comment isn't really relevant anymore. This *is* direct. Maybe just

```
// Finalize all id creation ranges before serializing. The production codepath does this
// in ContainerRuntime.createSummary.
```